dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/TrackerApiTest.java

```diff
@@ -27,40 +27,176 @@
  */
 package org.hisp.dhis.tracker;
 
+import java.io.File;
+import java.util.List;
+
 import org.hisp.dhis.ApiTest;
+import org.hisp.dhis.Constants;
 import org.hisp.dhis.actions.LoginActions;
-import org.hisp.dhis.actions.RestApiActions;
+import org.hisp.dhis.actions.MaintenanceActions;
 import org.hisp.dhis.actions.metadata.ProgramActions;
-import org.hisp.dhis.actions.tracker.EventActions;
-import org.hisp.dhis.actions.tracker.TEIActions;
+import org.hisp.dhis.actions.tracker.TrackerImportExportActions;
+import org.hisp.dhis.dto.TrackerApiResponse;
+import org.hisp.dhis.helpers.file.FileReaderUtils;
+import org.hisp.dhis.helpers.file.JsonFileReader;
+import org.hisp.dhis.tracker.imports.databuilder.RelationshipDataBuilder;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
 
+import com.google.gson.JsonObject;
+
 /**
  * @author Gintare Vilkelyte <vilkelyte.gintare@gmail.com>
  */
-@Tag( "category:tracker" )
+@Tag( "category:tracker_nti" )
```

Comment:
Yes, I'd like to change these tags at as well. These tags are related to https://test.tools.dhis2.org/reportportal/ui/#dhis2_auto/dashboard/2

I need to figure out how it would be affected, what I will need to change 😅 So I would suggest we do this separately.